ruby/rubocop.yml

```diff
@@ -198,3 +84,65 @@ Lint/NonLocalExitFromIterator:
   Enabled: false
 Lint/AmbiguousRegexpLiteral:
   Enabled: false
+
```

Comment:
Same. Initial set of rules opted out after playing a bit with `Rainforest` repo with the new configuration. 

The idea is to keep adding more rules to opt-out as we discover them to be annoying or not valuable.